src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -118,8 +118,8 @@ private string TryGetText(SyntaxToken token, SemanticModel semanticModel, Docume
                 TryGetTextForCombinationKeyword(token, syntaxFacts, out text) ||
                 TryGetTextForKeyword(token, syntaxFacts, out text) ||
                 TryGetTextForPreProcessor(token, syntaxFacts, out text) ||
-                TryGetTextForSymbol(token, semanticModel, document, cancellationToken, out text) ||
-                TryGetTextForOperator(token, document, out text))
+                TryGetTextForOperator(token, document, out text) ||
```

Comment:
Nit: The interpolated and verbatim string characters are not really operators are they? They are in the "Special characters" section of the docs, or the "Tokens" folder of the docs repo, so _maybe_ we should mirror that here?

I don't have strong feelings about this (though would have made this diff easier to read 😛)

Reply:
@davidwengier I've mode them to a new method.